install/kubernetes/helm/subcharts/kiali/values.yaml

```diff
@@ -20,6 +20,14 @@ ingress:
     # - secretName: kiali-tls
     #   hosts:
     #     - kiali.local
+service:
+  annotations: {}
+  name: http-kiali
```

Comment:
please delete the name - we are really working hard on removing stuff from values.yaml that does not absolutely need to be configured.  Think minimalism :)

Reply:
Fair enough, I was not aware of that. Fixed! :)